data/brands/office/estate_agent.json

```diff
@@ -129,6 +129,17 @@
         "office": "estate_agent"
       }
     },
+    {
+      "displayName": "Buxton",
+      "id": "buxton-d66f4e",
+      "locationSet": {"include": ["au"]},
```

Comment:
Is Buxton only in Victoria? If so, you could use "_au-vic.geojson_"

Reply:
Thanks, I'll keep that in mind from now on.